command.go

```diff
@@ -111,3 +156,11 @@ func (s *CommandSequence) Extend(cmds ...Command) *CommandSequence {
 	}
 	return s
 }
+
+func CommandFactory(cmdName string) (Command, error) {
```

Comment:
can we move all of the registry stuff into its own file. 

and change the name of this function to GetCommand(), and have the registry method have all of the logic here, so its just a straight wrapper?

Reply:
done